### ISSUE TITLE
Select the country via header or other param

### ISSUE THREAD

**Issue by botzill** [luminati-io/luminati-proxy#187]
Hi. 

Currently to select the country we use some the following format: `curl -x lum-country-[country-iso-code]:[zone-password]@[IP]:[port] [destination-URL]`. 

Is there a way to set the header as we have for `x-lpm-session`? Would be nice to have this feater as well. 

Thx.

**Comment by fudali** [comment]
Can you elaborate? Would you like to be able to choose country using x-lpm-country header instead of setting it in auth?

**Comment by botzill** [comment]
Yes @fudali, exactly that. it's possible?

**Comment by fudali** [comment]
Yes, it makes sense to me. But since it's not a bug rather a feature request and it's possible to achieve the same results with current functionality we can deliver this in lower priority

**Comment by botzill** [comment]
I see, thx. We have a scenario where we can't use auth method and having such a header would help us to integrate service into our platform. 

Thx a lot for your info and considering this as a feature.

**Comment by fudali** [comment]
I see. In the meantime you can contact our support team [here](https://luminati.io/faq#support) to find out a work-around or a temporary solution before we implement this. Please refer to this github issue as well.

**Comment by botzill** [comment]
Well, not sure if there is a workaround at the moment. The only way to pass county is via auth params, as I see from the docs.

Thx a lot!
